src/EA.Weee.Database/scripts/Update/Release 8/Sprint1/20250127-1256-alter-Evidence-getTransferNotes.sql

```diff
@@ -0,0 +1,67 @@
+﻿/****** Object:  StoredProcedure [Evidence].[getTransferNotes]    Script Date: 27/01/2025 12:42:23 ******/
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+ALTER PROCEDURE [Evidence].[getTransferNotes]
```

Comment:
It looks like the new columns should be added to TransferNoteData
 and GetTransferNotesReportHandler. Otherwise the Transfer download doesn't have Cat15

Reply:
now fixed.